mlir/lib/Dialect/LLVMIR/IR/LLVMDialect.cpp

```diff
@@ -1036,7 +1036,8 @@ void CallOp::build(OpBuilder &builder, OperationState &state, TypeRange results,
         /*op_bundle_operands=*/{}, /*op_bundle_tags=*/{},
         /*arg_attrs=*/nullptr, /*res_attrs=*/nullptr,
         /*access_groups=*/nullptr, /*alias_scopes=*/nullptr,
-        /*noalias_scopes=*/nullptr, /*tbaa=*/nullptr);
+        /*noalias_scopes=*/nullptr, /*tbaa=*/nullptr,
+        /*no_inline*/ nullptr, /*always_inline*/ nullptr);
```

Comment:
```suggestion
        /*no_inline=*/nullptr, /*always_inline=*/nullptr);
```
Nit: Same for all other instances.